pkg/csi/service/common/vsphereutil.go

```diff
@@ -32,13 +32,29 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/protobuf/types/known/timestamppb"
+
 	cnsvolume "sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/cns-lib/volume"
 	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/cns-lib/vsphere"
+	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/config"
 	csifault "sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/fault"
 	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/utils"
 	"sigs.k8s.io/vsphere-csi-driver/v2/pkg/csi/service/logger"
 )
 
+// VanillaCreateBlockVolParamsForMultiVC stores the parameters
+// required to call CreateBlockVolumeUtilForMultiVC function.
+type VanillaCreateBlockVolParamsForMultiVC struct {
+	Vcenter                   *vsphere.VirtualCenter
+	VolumeManager             cnsvolume.Manager
+	CNSConfig                 *config.Config
+	StoragePolicyID           string
+	Spec                      *CreateVolumeSpec
+	SharedDatastores          []*vsphere.DatastoreInfo
+	SnapshotDatastoreURL      string
+	ClusterFlavor             cnstypes.CnsClusterFlavor
+	FilterSuspendedDatastores bool
```

Comment:
Can we assume all already released features are enabled while adding new features?

Reply:
Done